Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -491,6 +491,14 @@ namespace AzToolsFramework
             }
         }
 
+        void AssetBrowserTreeView::mouseDoubleClickEvent(QMouseEvent* ev)
+        {
+            if (ev->button() == Qt::MouseButton::LeftButton)
+            {
+                emit doubleClickedSignal(QModelIndex());
```

Comment:
why have the model index as a signal parameter if you're always emitting a default constructed index? Do you have plans to use this signal more robustly in the future?

Reply:
Totally rewritten.